source/admin_manual/list_of_events.rst

```diff
@@ -205,10 +205,19 @@ These fields are common for the rest of the authentication client events.
 +--------------------+-----------------------------------------------------------+
 | Field             | Description                                                |
 +===================+============================================================+
+| mechanism         | Name of used SASL mechanism (e.g. PLAIN)                   |
++-------------------+------------------------------------------------------------+
```

Comment:
Looks like these were supposed to have been part of https://github.com/dovecot/documentation/pull/120 which was added for v2.3.13. I guess there should be some `.. versionadded:: v2.3.13` somewhere in here, beginning of the "common fields" I guess?..